.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-latest, ubuntu-latest]
```

Comment:
Is it not working for windows anymore?

Reply:
The [error](https://github.com/SAP/cloud-sdk-js/runs/2298458310?check_suite_focus=true) itself only happens with windows and says
```
Error: src/http-client/http-request-config.ts(1,24): error TS2305: Module '"../../../../node_modules/@sap-cloud-sdk/util/dist"' has no exported member 'exclude'.
```
The `exclude` was added later after the initial execution and was expected to be in the `node_modules`.
However, the cache was hit and the old `node_modules` was used, which did not know `exclude`.

Therefore, it's a combination of our cache and windows (that has different behaviour compared to other environment).

Reply:
By the way, the main build also failed only with windows: https://github.com/SAP/cloud-sdk-js/runs/2303633553?check_suite_focus=true. Retry did not work and it failed in the [PR](https://github.com/SAP/cloud-sdk-js/pull/1184) already before merging.
I guess we are not ready for introducing windows now, and it should be investigated in follow ups.

Reply:
That's ok for me, rerun did solve it on main now, but this should not be necessary (multiple times).